stdlib/public/core/UnicodeScalar.swift

```diff
@@ -249,7 +249,7 @@ public struct UnicodeScalar :
 extension UnicodeScalar : CustomStringConvertible, CustomDebugStringConvertible {
   /// An escaped textual representation of the Unicode scalar.
   public var description: String {
-    return String(value)
+    return escaped(asASCII: false)
```

Comment:
This used to return the character in quotes.  `escaped()` produces a non-quoted character.